includes/http/HTTPResponse.hpp

```diff
@@ -0,0 +1,29 @@
+#ifndef HTTP_RESPONSE_HPP
+#define HTTP_RESPONSE_HPP
+
+#include <map>
+#include <sstream>
+#include <string>
+
+namespace webserv {
+
+struct HTTPResponse {
```

Comment:
Is this just the temporary version?

Reply:
what do you mean by temporary?

Reply:
temporary
/ˈtɛmp(ə)r(ər)i/
adjective
lasting for only a limited period of time; not permanent.

Reply:
ah, ok. Thank you for clarifying :smiley: 

We can discuss it, but this was the way I was doing it before I saw the way you did your request.  What would you suggest?

Reply:
Ahahah, should it be a struct? Is there no need to have private attributes and getters and setters?

Reply:
yes, it should be a struct

Reply:
Alright, solving conversation then, looks good 👍